src/main/java/seedu/address/model/income/Income.java

```diff
@@ -0,0 +1,130 @@
+package seedu.address.model.income;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.claim.Amount;
+import seedu.address.model.claim.Description;
+import seedu.address.model.commonvariables.Name;
+import seedu.address.model.commonvariables.Phone;
+import seedu.address.model.tag.Tag;
+
+
+
```

Comment:
Perhaps you would like to remove these additional blank lines?

Reply:
Great suggestion! Thank you